public/js/app.js

```diff
@@ -0,0 +1,315 @@
+                var color1 = "rgb(" + r1 + "," + g1 + "," + b1 + ")";
+
+                var r2 = Math.round(istep * c1_0[0] + step * c1_1[0]);
+                var g2 = Math.round(istep * c1_0[1] + step * c1_1[1]);
+                var b2 = Math.round(istep * c1_0[2] + step * c1_1[2]);
+                var color2 = "rgb(" + r2 + "," + g2 + "," + b2 + ")";
+
+                $('.background').css({
+                    background: "-webkit-gradient(linear, left top, right top, from(" + color1 + "), to(" + color2 + "))"
+                }).css({
+                    background: "-moz-linear-gradient(left, " + color1 + " 0%, " + color2 + " 100%)"
+                });
+
+                step += gradientSpeed;
+                if (step >= 1) {
+                    step %= 1;
+                    colorIndices[0] = colorIndices[1];
+                    colorIndices[2] = colorIndices[3];
+
+                    colorIndices[1] = (colorIndices[1] + Math.floor(1 + Math.random() * (colors.length - 1))) % colors.length;
+                    colorIndices[3] = (colorIndices[3] + Math.floor(1 + Math.random() * (colors.length - 1))) % colors.length;
+                }
+            }
+
+            /**************** ScrollReveal *****************/
+            setInterval(updateGradient, 10);
+
+            window.sr = ScrollReveal({
+                reset: true
+            });
+            ScrollReveal().reveal('.headline', {
+                delay: 100
+            });
+            ScrollReveal().reveal('.tagline', {
+                delay: 500
+            });
+            ScrollReveal().reveal('.punchline', {
+                delay: 1000
+            });
+
+            sr.reveal('.sc-item', {
+                rotate: {
+                    x: 100,
+                    y: 100,
+                    z: 0
+                },
+                duration: 1000
+            });
+
+            /**************** Menu Hamburger *****************/
+
+            var content = document.querySelector('#hamburger-content');
+            var sidebarBody = document.querySelector('#hamburger-sidebar-body');
+
+            sidebarBody.innerHTML = content.innerHTML;
+
+
+            var button = document.querySelector('#hamburger-button');
+            var overlay = document.querySelector('#hamburger-overlay');
+            var activatedClass = 'hamburger-activated';
+
+            button.addEventListener('click', function (e) {
+                e.preventDefault();
+
+                this.parentNode.classList.add(activatedClass);
+            });
+
+            button.addEventListener('keydown', function (e) {
+                if (this.parentNode.classList.contains(activatedClass)) {
+                    if (e.repeat === false && e.which === 27)
+                        this.parentNode.classList.remove(activatedClass);
+                }
+            });
+
+            overlay.addEventListener('click', function (e) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
            overlay.addEventListener("click", function (e) {
```